### PR TITLE
Add more styling primitives

### PR DESCRIPTION
**Pull request by rudolfs**
We want to be able to apply all of the styling that @juliendonck provided.
* add title atom
* restructure files
  * introduce molecule namespace and add `topbar` and `footer` to it
  * move `link` to atoms
* fix font styling
* fix button styling and make its interface similar to title elements
* remove unneeded navigation elements

### CODE REVIEW

File: app/src/App.re [hunk 1]
```diff
@@ -1,11 +1,26 @@
-open Css;
+module Styles = {
+  open Css;
 
-global("body", [color(Particle.Color.black()), ...Particle.Font.text]);
+  global(
+    "body",
+    [
+      color(Particle.Color.black()),
+      unsafe("-webkit-font-smoothing", "antialiased"),
```
Comment (xla) [inline app/src/App.re]: The dark side is strong in this one!

File: app/src/App.re [hunk 2]
```diff
@@ -1,11 +1,26 @@
...
+      unsafe("-moz-osx-font-smoothing:", "grayscale"),
+      ...Particle.Font.text,
+    ],
+  );
+
+  global("a", [color(Particle.Color.black()), textDecoration(none)]);
```
Comment (xla) [inline app/src/App.re]: Can we add `cursor: pointer` to indicate that they are clickable, trips me up even in local debug. :>

File: app/src/App.re [hunk 3]
```diff
@@ -1,11 +1,26 @@
...
+
+  let app = style([]);
```
Comment (xla) [inline app/src/App.re]: Is this intentionally empty?

File: app/src/App.re (hunk 1 above)
Comment (rudolfs) [reply app/src/App.re]: 👹

File: app/src/App.re (hunk 3 above)
Comment (rudolfs) [reply app/src/App.re]: Yeah, forgot to remove it.

**Pull request merged**